docs/sources/developers/plugins/migration-guide.md

```diff
@@ -4,77 +4,451 @@ title = "Plugin migration guide"
+
+For instructions on plugin signing, refer to the [Plugin signatures](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/#sign-a-plugin) topic.
+
+You can still run and develop an unsigned plugin by running your Grafana instance in [development mode](https://grafana.com/docs/grafana/latest/administration/configuration/#app_mode).
+
+### Time series data can now be structured in a wide or many format.
+
+In Grafana 8.x, you can structure time series data in either `wide` or `many` formats. If your plugin supports time series data, we recommend that you support both formats.
+
+Grafana supported only the `many` format before. For example:
+
+| time                 | temperature |
+| -------------------- | ----------- |
+| 2021-05-08T07:31:45Z | 23          |
+| 2021-05-08T09:31:45Z | 25          |
+| 2021-05-08T11:31:45Z | 27          |
 
-It's written for:
+| time                 | humidity |
+| -------------------- | -------- |
+| 2021-05-08T07:31:45Z | 60       |
+| 2021-05-08T09:31:45Z | 55       |
+| 2021-05-08T11:31:45Z | 55       |
 
-- Plugin authors who want to migrate their plugins to Grafana 7.0+.
-- Plugin users who are using custom plugins and want to know whether they can upgrade to Grafana 7.0 without losing functionality.
+> Data is delivered as an array of data frames where the `time` field is repeated.
 
-> If you've successfully migrated your plugin from Angular to React, please [submit an issue on GitHub](https://github.com/grafana/grafana/issues/new?title=Docs%20feedback:%20/developers/plugins/migration-guide.md) and share your experiences with us so that we can improve this guide!
+That made it possible to detect time series data by inspecting the data frame. If it had two fields (time + value) it most likley was time series data. This is no longer possible since the same data can be delivered in the following `wide` format.
 
-## What's new in Grafana 7.0?
+| time                 | temperature | humidity |
+| -------------------- | ----------- | -------- |
+| 2021-05-08T07:31:45Z | 23          | 60       |
+| 2021-05-08T09:31:45Z | 25          | 55       |
+| 2021-05-08T11:31:45Z | 27          | 55       |
+
+> Data delivered as one single data frame (the values share the same time field).
+
+If your plugin should support time series data we recommend you to make sure to support both.
+
+### Update react-hook-form from v6 to v7
+
+We have upgraded react-hook-form from version 6 to version 7. We recommend following the [react-hook-form-migration-guide](https://react-hook-form.com/migrate-v6-to-v7/) to make your forms compatible with version 7.
+
+### Update the plugin.json
+
+The property that defines which Grafana version your plugin supports has been renamed.
+
+```json
+// before
+{
+"dependencies": {
+    "grafanaVersion": "7.5.x",
+    "plugins": []
+  }
+}
+
+// after
+{
+  "dependencies": {
+    "grafanaDependency": "8.0.0",
```

Comment:
The format should be `grafanaDependency: '>=8.0.0`

Also,  It seems the plugin website still uses the `grafanaVersion` as shown in the [servicenow datasource documentation](https://grafana.com/grafana/plugins/grafana-servicenow-datasource/) whereas the saphana plugin.json uses `grafanaDependency` (new property) but the [plugin website](https://grafana.com/grafana/plugins/grafana-saphana-datasource/) doesn't honor that. It seems we need to fix the website to show the correct dependency version from `grafanaDependency`.

Reply:
Yes, great feedback! Will look into this 🙏🏻

Reply:
Exactly, it seems like the website template is only looking for the `grafanaVersion` property at the moment, which is not correct. ([Source code](https://github.com/grafana/website/blob/7fb3e6dbdffef62f1f0f5582581b4492091ccc66/layouts/grafana/plugin.html#L100))

I think it should be taking the `grafanaDependency` and falling back to `grafanaVersion` for backward compatibility.
Will create an issue for this.

**Also updated the code sample in the migration guide to use a range.**